serialization/json_value.go

```diff
@@ -16,8 +16,32 @@
 
 package serialization
 
+import "errors"
+
 type JSON []byte
 
 func (j JSON) String() string {
 	return string(j)
 }
+
+// MarshalJSON is used by functions in the Go `encoding/json` package. This
+// function ensures that a `serialization.JSON` value is correctly serialized
```

Comment:
Could you move "This" after the first sentence to the line below? Similary for other comment lines. I think it would be great if each sentence is in its own line (no matter the length of the line).

Reply:
Addressed in ab9de482c6c8c3a68a0c468890c7022da27ea8f6